sdk/storage/azure_storage_blob_test/src/lib.rs

```diff
@@ -50,53 +62,27 @@ pub fn get_blob_service_client(recording: &Recording) -> Result<BlobServiceClien
 /// # Arguments
 ///
 /// * `recording` - A reference to a Recording instance.
-pub fn get_container_client(recording: &Recording) -> Result<BlobContainerClient> {
-    let container_name = recording
-        .random_string::<17>(Some("container"))
-        .to_ascii_lowercase();
+/// * `create` - An optional flag to determine whether the container should also be created.
+pub async fn get_container_client(
+    recording: &Recording,
+    create: Option<bool>,
```

Comment:
Since Rust really doesn't have optional params and therefore we need to always pass something here, even if that is just `None`, this should probably just be a required bool. In all cases you use it, you are passing `Some` anyway.